papaya-core/src/main/java/pl/edu/agh/papaya/service/sprint/SprintCreationWizard.java

```diff
@@ -0,0 +1,60 @@
+package pl.edu.agh.papaya.service.sprint;
+
+import java.time.Duration;
+import java.util.Objects;
+import pl.edu.agh.papaya.model.LocalDateTimePeriod;
+import pl.edu.agh.papaya.model.Project;
+import pl.edu.agh.papaya.model.Sprint;
+import pl.edu.agh.papaya.util.AssertionUtil;
+
+@SuppressWarnings({"checkstyle:HiddenField", "PMD.BeanMembersShouldSerialize"})
+public class SprintCreationWizard {
+
+    private final SprintService sprintService;
+
+    private Project project;
+    private LocalDateTimePeriod enrollmentPeriod;
+    private LocalDateTimePeriod durationPeriod;
+    private Duration timePlanned;
+
+    SprintCreationWizard(SprintService sprintService) {
+        this.sprintService = Objects.requireNonNull(sprintService);
+    }
+
+    public SprintCreationWizard withProject(Project project) {
+        this.project = Objects.requireNonNull(project);
+        return this;
+    }
+
+    public SprintCreationWizard withEnrollmentPeriod(LocalDateTimePeriod enrollmentPeriod) {
+        this.enrollmentPeriod = Objects.requireNonNull(enrollmentPeriod);
+        return this;
+    }
+
+    public SprintCreationWizard withDurationPeriod(LocalDateTimePeriod durationPeriod) {
+        this.durationPeriod = Objects.requireNonNull(durationPeriod);
+        return this;
+    }
+
+    public SprintCreationWizard withTimePlanned(Duration timePlanned) {
+        this.timePlanned = Objects.requireNonNull(timePlanned);
+        return this;
+    }
+
+    public Sprint create() {
```

Comment:
Nie wiem, jak robimy z pokrywającymi się sprintami - nie jestem pewna, czy są jakieś use case'y, w których to mogłoby być przydatne, a w sumie tego chyba nie sprawdzamy. Overlapujące się sprinty w sumie nas nie bolą, bo w widoku Excela zawsze będzie najbliższe niezamknięte, ale zastanawiam się, jak z pokrywającymi się zapisami - nie wiem, czy teraz powstaną strzałki umożliwiające przejrzenie wszystkich sprintów `DECLARABLE`, ale jak nie, to lepiej byłoby to tutaj wymusić